gettsim/soz_vers/krankenv_pflegev.py

```diff
@@ -3,71 +3,72 @@
 
 
 def ges_krankenv_beitr_m(
-    geringfügig_beschäftigt,
+    _geringfügig_beschäftigt,
     ges_krankenv_beitr_rente,
     ges_krankenv_beitr_selbst,
-    krankenv_beitr_regulär_beschäftigt,
+    krankenv_beitr__regulär_beschäftigt,
     an_beitr_krankenv_midi_job,
 ):
 
-    ges_krankenv_beitr_m = pd.Series(
-        index=geringfügig_beschäftigt.index, name="ges_krankenv_beitr_m", dtype=float
+    out = pd.Series(
+        index=_geringfügig_beschäftigt.index, name="ges_krankenv_beitr_m", dtype=float
     )
 
-    ges_krankenv_beitr_m.loc[geringfügig_beschäftigt] = 0
+    out.loc[_geringfügig_beschäftigt] = 0
 
     # Assign calculated contributions
-    ges_krankenv_beitr_m.loc[
-        an_beitr_krankenv_midi_job.index
-    ] = an_beitr_krankenv_midi_job
-    ges_krankenv_beitr_m.loc[
-        krankenv_beitr_regulär_beschäftigt.index
-    ] = krankenv_beitr_regulär_beschäftigt
-    ges_krankenv_beitr_m.loc[
-        ges_krankenv_beitr_selbst.index
-    ] = ges_krankenv_beitr_selbst
+    out.loc[an_beitr_krankenv_midi_job.index] = an_beitr_krankenv_midi_job
+    out.loc[
+        krankenv_beitr__regulär_beschäftigt.index
+    ] = krankenv_beitr__regulär_beschäftigt
+    out.loc[ges_krankenv_beitr_selbst.index] = ges_krankenv_beitr_selbst
```

Comment:
For combining multiple exclusive series, I would use `.append()`. The where part might not work.
```python
out = _geringfügig_beschäftigt.astype(int)
others = an_beitr_krankenv_midi_job.append(krankenv_beitr__regulär_beschäftigt)
return out.where(_geringfügig_beschäftigt, others) + ges_krankenv_beitr_rente
```

Reply:
What's the advantage of `append` over the other way. I think for users it is more readable the way it is right now!